src/tests/unit/components/Envelope.vue.spec.js

```diff
@@ -80,7 +81,7 @@ describe('Envelope', () => {
 			localVue,
 		})
 
-		expect(view.vm.hasSeenAcl).toBe(true)
+		expect(mailboxHasRights(view.vm.mailbox,'s')).toBe(true)
```

Comment:
if the component no longer has a `hasSeenAcl` drop this test

if there is a `hasSeenAcl` that calls `mailboxHasRights` keep the test and test `hasSeenAcl` like before